src/app/map/map.component.html

```diff
@@ -1 +1,6 @@
-<div id="map"></div>
\ No newline at end of file
+<div id="map">
+    <div class="leaflet-bottom leaflet-left">
+        <span>Zoom Level: {{currentZoom}}</span> <br />
+        <span>Lat: {{currentLat.toFixed(4)}}, Lng: {{currentLng.toFixed(4)}} </span>
+    </div>
```

Comment:
The team decided earlier on that we'll just include map scale in this bottom left area, at least for now. That's included in issue #10, so you can omit this for now. Looks good though!